network-api/networkapi/wagtailpages/templates/fragments/most_voted_rating.html

```diff
@@ -0,0 +1,34 @@
+{% load i18n %}
+
+{% comment %}
+ This is where we render the "creepiness face" icon on a product's page. We use the following keys:
+ 0 = Not creepy
+ 1 = A little creepy
+ 2 = Somewhat creepy
+ 3 = Very creepy
+ 4 = Super creepy
+{% endcomment %}
```

Comment:
With the "vote information" functions added to ProductPageVotes, we now don't need this comment, and the template drasticaly simplifies to just:

```
<span class="body-small">
    {% with vote_data=product.votes.get_most_voted %}
    <a class="body-small" href="#creepiness-vote">{% trans "People voted" %}</a>:
    <span class="people-voted {{ vote_data.label|slugify }}">{{ vote_data.localized }}</span>
    {% endwith %}
</span>
```

Reply:
Great suggestion!